app/models/donation.rb

```diff
@@ -181,4 +197,38 @@ def subscribe_attributes
     return_attributes[:CITY] = self.activist.city if self.activist and self.activist.city
     return_attributes
   end
+
+  def notify_activist(template_name, template_vars = {}, auto_deliver = true)
+    Notification.notify!(
+      activist_id,
+      template_name,
+      default_template_vars.merge(template_vars),
+      community_id,
```

Comment:
you are right we does not have the community_id just need to use comunity.id, send a commit about this already too